SYCL/Basic/query.cpp

```diff
@@ -0,0 +1,41 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env CreateMultipleSubDevices=2 env EnableTimestampPacket=1 \
```

Comment:
I thought the CreateMultipleSubDevices=2 would not work on DG1/ATS with 1 tile
What kind of GPU HW do we tests with ?

Reply:
@smaslov-intel, @glyons-intel  We are using GEN9 machines. But the test can be run on DG1/ATS as well. Please mark as UNSUPPORTED: gpu-intel-dg1 to make sure that the test will not fail there (if it is exp[ected behavior) or XFAIL: gpu-intel-dg1 (if it is going to be fixed).

Reply:
Done.